azurerm/internal/services/mssql/mssql_database_resource.go

```diff
@@ -123,7 +122,7 @@ func resourceArmMsSqlDatabase() *schema.Resource {
 				Type:         schema.TypeFloat,
 				Optional:     true,
 				Computed:     true,
-				ValidateFunc: azValidate.FloatInSlice([]float64{0.5, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.25, 2.5, 3, 4, 5}),
+				ValidateFunc: validate.MsSqlDatabaseMinCapacity,
```

Comment:
can we add an [Acceptance Test](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/internal/services/mssql/tests/mssql_database_resource_test.go#L16-L33) to provision a Database with the Capacity of `0` to confirm the Basic SKU / #8226 is fixed